src/commands/createNewProject/NewProjectLanguageStep.ts

```diff
@@ -45,16 +44,18 @@ export class NewProjectLanguageStep extends AzureWizardPromptStep<IProjectWizard
             { label: ProjectLanguage.JavaScript },
             { label: ProjectLanguage.TypeScript },
             { label: ProjectLanguage.CSharp },
+            { label: ProjectLanguage.PowerShell, description: previewDescription },
             { label: ProjectLanguage.Python, description: previewDescription },
             { label: ProjectLanguage.Java }
         ];
 
-        if (getFuncExtensionSetting('enablePowerShell')) {
```

Comment:
Double check with @anirudhgarg  that we should be doing this yet. (cc @joeyaiello)

Reply:
I've checked with @anirudhgarg and @eamonoreilly, and given that the next release will be at the beginning of May, they are fine with the change. (cc @joeyaiello)

Reply:
I prefer to keep master branch "release ready", so let's hold off on removing the feature flag. Plans change - we might release early or PowerShell support might release late. It's easy enough to remove the flag a day or two before release

Reply:
As another reason to delay removal of the feature flag, we have [this issue](https://github.com/Microsoft/vscode-azurefunctions/issues/1082) tracking it and ideally all of the bullet points are done first

Reply:
@EricJizbaMSFT : All the back end work is done. Managed Dependencies is already available in the Portal, and will be available in the next CLI release.

Reply:
@Francisco-Gamino the feature flag isn't related to managed dependencies, though. It's there because PowerShell is still in private preview in Azure (Correct? I don't see the runtime option when creating a function app) and because the items listed in [this issue](https://github.com/Microsoft/vscode-azurefunctions/issues/1082) have not been done yet.

![Screen Shot 2019-04-10 at 9 48 53 AM](https://user-images.githubusercontent.com/11282622/55897781-13533700-5b76-11e9-8dec-fe94a7d17c5f.png)

Reply:
Thanks @EricJizbaMSFT for checking. PowerShell is supposed to be available for Public Preview in the first week of May. The Host (runtime), PowerShell worker and CLI changes are done. I am trying to finish the VSCode extension work as I am going on vacation starting 4/12. I will let Eamon and Anirud comment on the Portal changes.

@eamonoreilly @anirudhgarg: when is PowerShell going to be available as part of the runtime options?

Reply:
I'm happy to remove the feature flag myself closer to our release date if you'll be OOF. I want to make sure all the docs/tests are up to par with our other languages before that happens. We already have the above issue in our current milestone, so it won't be forgotten.

On the other hand I'm not as familiar with the managed dependency stuff so I would rather get this PR in before you're OOF.